packages/loot-core/src/server/cloud-storage.ts

```diff
@@ -279,7 +279,7 @@ export async function upload() {
     console.log('Upload failure', err);
 
     if (err instanceof PostError) {
-      throw new FileUploadError(
+      throw FileUploadError(
```

Comment:
Just double-checking: will this actually work?

Reply:
That function is a misnomer as it returns an object, not an error. Also it is used like that (without `new`) in few other places, so I guess it is fine.